src/main/java/org/openrefine/extensions/commons/utils/CommonsImportingController.java

```diff
@@ -212,19 +328,18 @@ public void run() {
                     pm.setName(JSONUtilities.getString(optionObj, "projectName", "Untitled"));
                     pm.setEncoding(JSONUtilities.getString(optionObj, "encoding", "UTF-8"));
 
-                    /*try {
-                        /*parsePreview(
-                            databaseQueryInfo,
-                            job.project,
-                            job.metadata,
+                    try {
+                        parsePreview(
```

Comment:
Intuitively we should not be calling `parsePreview` in `doCreateProject` (preview and project creation are distinct phases), see the first comment above.